### PR TITLE
10 post api articles articleid comments

### PR DESCRIPTION
**Pull request by laurie60**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -250,4 +250,57 @@ describe("News Express App", () => {
...
+        .post("/api/articles/potato/comments")
+        .send(comment)
+        .expect(400)
+        .then(({ body }) => {
+          expect(body).toEqual({
+            msg: `invalid input type`,
+          });
+        });
+    });
+  });
```
Comment (Rayhaan2k9) [inline __tests__/app.test.js:322]: The post body should have a key of username and body like {username: "icellusedcars", body: "Never gonna give you up"} Also I would add tests for if the post has missing info like if the username or body is missing

**Review by Rayhaan2k9 — CHANGES_REQUESTED** [review]
Almost there. Just check my comment :)

File: __tests__/app.test.js [hunk 2]
```diff
@@ -250,4 +250,74 @@ describe("News Express App", () => {
...
+        .send(comment)
+        .expect(400)
+        .then(({ body }) => {
+          expect(body).toEqual({
+            msg: `invalid input type`,
+          });
+        });
+    });
+
+    test("400: with comment of invalid input type if input is missing username ", () => {
```
Comment (karlaferry) [inline __tests__/app.test.js:309]: You also want to test that `body` exists in the post body :) So just make another test that looks like this but this time handle it if `body` is missing.

**Review by karlaferry — APPROVED** [review]
Approving but make sure to add that last test! Also, well done, very nice use of async/await 👏🏼

**Pull request merged**